riemann_client/client.py

```diff
@@ -24,8 +24,8 @@ def emit(self, record):
     Timer = None
 import time
 
-import riemann_client.riemann_pb2
-import riemann_client.transport
+from . import riemann_pb2
+from .transport import UDPTransport, TCPTransport
```

Comment:
I initially wanted to keep it as `from . import transport`, but that would then conflict with the `transport` variable in `Client.__init__()`. I hope you can look past the small namespace pollution I'm introducing.